src/transformers/modeling_utils.py

```diff
@@ -2142,7 +2146,10 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             logger.info("Detected 8-bit loading: activating 8-bit loading for this model")
 
             # We never convert lm_head or any last modules for numerical stability reasons
```

Comment:
as you added in the docs, the above comment is outdated wrt "last" - it doesn't have to be last (jukebox)

Reply:
Thanks! Fixed in 224b504fc08506e5b8cafa164c4b8bcad454b81e